BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -354,10 +354,43 @@
         ]]>
     </string>
 
+    <string name="search_help_text">
+        <![CDATA[
+        <h1><u>Hilfe zur Suche</u></h1>
+        <p>Sie befinden sich in der Suche. Hier können Sie nach Regalen, Büchern und Notizen suchen.
+        </p>
+
+        <h2>Sortieren</h2>
+        <p>Sie können ein Kriterium, nach dem Sie ihre Suchergebnisse sortieren wollen, auswählen.
+        Dabei können Sie ihre Suchergebnisse absteigend oder aufsteigend nach Namen sortieren.
+        Außerdem können Sie die Suchergebnisse nach dem neuesten oder ältesten Veränderungsdatum
+        sortieren. </p>
+
+        <h2>Filtern</h2>
+        <p>Sie können ihre Suchergebnisse nach Regalen, Büchern und Notizen filtern.
+        Klicken Sie dazu die entsprechenden Optionen an, nach der Sie ihre Suchergebnisse
+        filtern wollen.</p>
+
+        ]]>
+    </string>
 
+    <!-- Search Fragment -->
+    <string name="search">Suchen …</string>
+    <string name="search_hint">Suche Regal, Buch oder Notiz</string>
```

Comment:
```suggestion
    <string name="search_hint">Suche nach einem Regal, einem Buch oder einer Notiz</string>
```

Reply:
This is too long so the hint is not fully displayed in the search. I will not change this.